guidelines/prd_template.md

```
@@ -0,0 +1,66 @@
+To get started with this template:
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)